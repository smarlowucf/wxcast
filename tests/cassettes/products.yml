interactions:
- request:
    body: null
    headers:
      Accept: ['*/*']
      Accept-Encoding: ['gzip, deflate']
      Connection: [keep-alive]
      User-Agent: ['wxcast app. https://github.com/smarlowucf/wxcast']
    method: GET
    uri: https://api.weather.gov/products/locations/BOU/types
  response:
    body: {string: "{\n    \"@context\": [\n        \"https://raw.githubusercontent.com/geojson/geojson-ld/master/contexts/geojson-base.jsonld\"\
        ,\n        {\n            \"@vocab\": \"https://api.weather.gov/ontology#\"\
        \n        }\n    ],\n    \"type\": \"FeatureCollection\",\n    \"features\"\
        : [\n        {\n            \"productCode\": \"AFD\",\n            \"productName\"\
        : \"Area Forecast Discussion\"\n        },\n        {\n            \"productCode\"\
        : \"FDI\",\n            \"productName\": \"Fire Danger Indices\"\n       \
        \ },\n        {\n            \"productCode\": \"FWF\",\n            \"productName\"\
        : \"Routine Fire Wx Fcst (With/Without 6-10 Day Outlook)\"\n        },\n \
        \       {\n            \"productCode\": \"FWL\",\n            \"productName\"\
        : \"Land Management Forecasts\"\n        },\n        {\n            \"productCode\"\
        : \"FWM\",\n            \"productName\": \"Miscellaneous Fire Weather Product\"\
        \n        },\n        {\n            \"productCode\": \"FWN\",\n         \
        \   \"productName\": \"Fire Weather Notification\"\n        },\n        {\n\
        \            \"productCode\": \"FWO\",\n            \"productName\": \"Fire\
        \ Weather Observation\"\n        },\n        {\n            \"productCode\"\
        : \"FWS\",\n            \"productName\": \"Suppression Forecast\"\n      \
        \  },\n        {\n            \"productCode\": \"HML\",\n            \"productName\"\
        : \"AHPS XML\"\n        },\n        {\n            \"productCode\": \"HRR\"\
        ,\n            \"productName\": \"Weather Roundup\"\n        },\n        {\n\
        \            \"productCode\": \"HWO\",\n            \"productName\": \"Hazardous\
        \ Weather Outlook\"\n        },\n        {\n            \"productCode\": \"\
        OPU\",\n            \"productName\": \"Other Public Products\"\n        },\n\
        \        {\n            \"productCode\": \"OSO\",\n            \"productName\"\
        : \"Other Surface Observations\"\n        },\n        {\n            \"productCode\"\
        : \"PFM\",\n            \"productName\": \"Point Forecast Matrices\"\n   \
        \     },\n        {\n            \"productCode\": \"PNS\",\n            \"\
        productName\": \"Public Information Statement\"\n        },\n        {\n \
        \           \"productCode\": \"RR2\",\n            \"productName\": \"Hydro-Met\
        \ Data Report Part 2\"\n        },\n        {\n            \"productCode\"\
        : \"RR3\",\n            \"productName\": \"Hydro-Met Data Report Part 3\"\n\
        \        },\n        {\n            \"productCode\": \"RRS\",\n          \
        \  \"productName\": \"HADS Data\"\n        },\n        {\n            \"productCode\"\
        : \"RTP\",\n            \"productName\": \"Regional Max/Min Temp and Precipitation\
        \ Table\"\n        },\n        {\n            \"productCode\": \"SRG\",\n\
        \            \"productName\": \"Soaring Guidance\"\n        },\n        {\n\
        \            \"productCode\": \"STQ\",\n            \"productName\": \"Spot\
        \ Forecast Request\"\n        },\n        {\n            \"productCode\":\
        \ \"SYN\",\n            \"productName\": \"Regional Weather Synopsis\"\n \
        \       },\n        {\n            \"productCode\": \"TVL\",\n           \
        \ \"productName\": \"Travelers Forecast\"\n        },\n        {\n       \
        \     \"productCode\": \"VFT\",\n            \"productName\": \"Terminal Aerodrome\
        \ Forecast (TAF) Verification\"\n        },\n        {\n            \"productCode\"\
        : \"ZFP\",\n            \"productName\": \"Zone Forecast Product\"\n     \
        \   }\n    ]\n}"}
    headers:
      Access-Control-Allow-Origin: ['*']
      Cache-Control: ['max-age=120, public, s-maxage=120']
      Connection: [keep-alive]
      Content-Length: ['3013']
      Content-Type: [application/geo+json]
      Date: ['Fri, 01 Dec 2017 02:08:25 GMT']
      Server: [nginx]
      Strict-Transport-Security: [max-age=31536000; includeSubDomains; preload]
      Vary: [Accept]
      X-Correlation-ID: [5baa7265-1cd3-4cc9-952f-ecc3b7831f7c]
      X-Request-ID: [5baa7265-1cd3-4cc9-952f-ecc3b7831f7c]
      X-Server-ID: [vm-bldr-nids-apiapp236.ncep.noaa.gov]
    status: {code: 200, message: OK}
version: 1
