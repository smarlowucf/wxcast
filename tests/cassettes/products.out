AFD:  Area Forecast Discussion
FDI:  Fire Danger Indices
FWF:  Routine Fire Wx Fcst (With/Without 6-10 Day Outlook)
FWL:  Land Management Forecasts
FWM:  Miscellaneous Fire Weather Product
FWN:  Fire Weather Notification
FWO:  Fire Weather Observation
FWS:  Suppression Forecast
HML:  AHPS XML
HRR:  Weather Roundup
HWO:  Hazardous Weather Outlook
OPU:  Other Public Products
OSO:  Other Surface Observations
PFM:  Point Forecast Matrices
PNS:  Public Information Statement
RR2:  Hydro-Met Data Report Part 2
RR3:  Hydro-Met Data Report Part 3
RRS:  HADS Data
RTP:  Regional Max/Min Temp and Precipitation Table
SRG:  Soaring Guidance
STQ:  Spot Forecast Request
SYN:  Regional Weather Synopsis
TVL:  Travelers Forecast
VFT:  Terminal Aerodrome Forecast (TAF) Verification
ZFP:  Zone Forecast Product
